core/project_settings.cpp

```diff
@@ -1085,6 +1085,10 @@ ProjectSettings::ProjectSettings() {
 
 	GLOBAL_DEF("audio/default_bus_layout", "res://default_bus_layout.tres");
 	custom_prop_info["audio/default_bus_layout"] = PropertyInfo(Variant::STRING, "audio/default_bus_layout", PROPERTY_HINT_FILE, "*.tres");
+	GLOBAL_DEF_RST("audio/general/2d_panning_strength", 1.0f);
+	custom_prop_info["audio/general/2d_panning_strength"] = PropertyInfo(Variant::REAL, "audio/general/2d_panning_strength", PROPERTY_HINT_RANGE, "0,4,0.01");
+	GLOBAL_DEF_RST("audio/general/3d_panning_strength", 1.0f);
+	custom_prop_info["audio/general/3d_panning_strength"] = PropertyInfo(Variant::REAL, "audio/general/3d_panning_strength", PROPERTY_HINT_RANGE, "0,4,0.01");
```

Comment:
In `3.x` the Audio settings are all top-level (`audio/name`) so the `/general` category should be removed (here, in the docs and where the settings are queried).

Reply:
Thank you. I changed paths of these settings.